examples/provider/provider.tf

```diff
@@ -0,0 +1,48 @@
+# This example creates an inventory named `My new inventory`
+# and adds a host `tf_host` and a group `tf_group` to it.
+#
+terraform {
+  required_providers {
+    aap = {
+      source = "ansible/aap"
+    }
+  }
+}
+
+provider "aap" {
+  host                 = "https://localhost:8043"
+  username             = "ansible"
+  password             = "test123!"
+  insecure_skip_verify = true
+}
+
+resource "aap_inventory" "my_inventory" {
+  name        = "My new inventory"
+  description = "A new inventory for testing"
+  variables = jsonencode(
+    {
+      "foo" : "bar"
+    }
+  )
+}
+
+resource "aap_host" "my_host" {
+  inventory_id = aap_inventory.my_inventory.id
+  name         = "tf_host"
+  variables = jsonencode(
+    {
+      "foo" : "bar"
+    }
+  )
+  groups = [2, 3, 4]
+}
+
+resource "aap_group" "my_group" {
```

Comment:
I guess if would be nice to have the example like:
- Create group
- Pass the id of the created group to the host into groups.